docs/DEVGUIDE.md

```diff
@@ -174,137 +178,248 @@ The images are tagged with the current Git commit SHA:
+  resourceVersion: "9"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/instances/ups-instance
+  uid: 8654e626-ffc9-11e6-b111-0242ac110005
+spec:
+  osbGuid: 34c984e1-4626-4574-8a95-9e500d0d48d3
+  planName: default
+  serviceClassName: user-provided-service
+status:
+  conditions:
+  - message: The instance was provisioned successfully
+    reason: ProvisionedSuccessfully
+    status: "True"
+    type: Ready
+```
+
+### Bind to the Instance
+
+Now that our `Instance` has been created, let's bind to it.  To do this, we
+create a new [`Binding`](../contrib/examples/walkthrough/ups-binding.yaml).
+
+    kubectl create -f contrib/examples/walkthrough/ups-binding.yaml
+    binding "ups-binding" created
+
+We can check the status of the `Instance` using `kubectl get`:
+
+    kubectl get bindings -n test-ns ups-binding -o yaml
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  creationTimestamp: 2017-03-07T01:44:36Z
+  finalizers:
+  - kubernetes
+  name: ups-binding
+  namespace: test-ns
+  resourceVersion: "29"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/bindings/ups-binding
+  uid: 9eb2cdce-02d7-11e7-8edb-0242ac110005
+spec:
+  instanceRef:
+    name: ups-instance
+  osbGuid: b041db94-a5a0-41a2-87ae-1025ba760918
+  secretName: my-secret
+status:
+  conditions:
+  - message: Injected bind result
+    reason: InjectedBindResult
+    status: "True"
+    type: Ready
+```
+
+Notice that the status has a ready condition set.  This means our binding is
+ready to use.  If we look at the secrets in our `test-ns` namespace in
+kubernetes, we should see:
+
+    kubectl get secrets -n test-ns
+    NAME                  TYPE                                  DATA      AGE
+    default-token-3k61z   kubernetes.io/service-account-token   3         29m
+    my-secret             Opaque                                2         1m
+
+Notice that a secret named `my-secret` has been created in our namespace.
+
+### Unbind from the Instance
+
+Now, let's unbind from the Instance.  To do this, we just delete the `Binding`
+that we created:
+
+    kubectl delete -n test-ns bindings ups-binding
+
+If we check the secrets in the `test-ns` namespace, we should see that the
+secret we were injected with has been deleted:
 
-    kubectl create -f contrib/examples/walkthrough/ups-backend.yaml
+    kubectl get secrets -n test-ns
+    NAME                  TYPE                                  DATA      AGE
+    default-token-3k61z   kubernetes.io/service-account-token   3         30m
 
-Now you can use the same steps as above to create a binding and a consuming
-service.
+### Deprovision the Instance
+
+Now, we can deprovision the instance.  To do this, we just delete the `Instance`
+that we created:
+
+    kubectl delete -n test-ns instances ups-instance
+
+### Delete the broker
+
+Now, we can delete the broker that we added.  When we do this, all the services
```

Comment:
We should reword this slightly. As currently stated this kind of implies that deleting brokers is part of a normal developer's workflow, when its really more of an admin feature and not used very often.  What about:

```
When the services managed by a broker need to be removed from the
Kubernetes platform we accomplish this by deleting the `broker` resource.
```